packages/renderer/vite.tests.setup.js

```diff
@@ -17,3 +17,5 @@
  ***********************************************************************/
 
 import 'vitest-canvas-mock';
+
+global.window.matchMedia = () => {};
```

Comment:
question: should it be `vi.fn()` rather than `() => {}`; as we're in the mocking parts

Reply:
It is never mocked in tests directly, so it does not matter IMO.

Reply:
Merged this commit to new dependabot pr for svelte@5.12.0